ansys/dpf/core/data_tree.py

```diff
@@ -25,7 +25,8 @@
 
 
 class DataTree:
-    """Represents an entity mapping attributes names to values.
+    """
```

Comment:
@cbellot000 I think there is a discrepancy between what flake8 or sphinx reports and what codacy reports regarding multiline docstrings beginning with an empy line. I think this version of writing it triggers a warning in one of the three.

Reply:
@PProfizi I fixed it for codacy

Reply:
reverted now